<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="display">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="content">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="maximumSize">
           <size>
            <width>225</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>165</width>
             <height>355</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="receiptName">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>30</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Vitajte v kuch√°rke</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
                <underline>false</underline>
               </font>
              </property>
              <property name="text">
               <string>Ingrediencie:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QLabel" name="receipt_ingredients">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="text">
               <string>Ingrediencie</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="10" column="0" colspan="2">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="11" column="0" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Postup:</string>
              </property>
             </widget>
            </item>
            <item row="12" column="1">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="13" column="0" colspan="2">
             <widget class="QLabel" name="receipt_content">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Postup</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="receipt_image">
              <property name="text">
               <string>Obrazok</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="edit_receipts">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="treeViewEdit">
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>477</width>
             <height>609</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="edit_mode_name">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>30</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Vyber recept</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="edit_mode_picture">
              <property name="text">
               <string>obrazok</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_mode_choose_picture">
              <property name="text">
               <string>Prehƒæad√°va≈•...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Ingrediencie</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="edit_mode_ingredients"/>
            </item>
            <item>
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Postup</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="edit_mode_content"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Ulo≈æi≈•</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSubor">
    <property name="title">
     <string>S√∫bor</string>
    </property>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRecepty">
    <property name="title">
     <string>Zobrazi≈•</string>
    </property>
    <addaction name="actionViewMode"/>
    <addaction name="separator"/>
    <addaction name="actionEditMode"/>
   </widget>
   <widget class="QMenu" name="menuPomocn_k">
    <property name="title">
     <string>Pomocn√≠k</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Upravi≈•</string>
    </property>
    <addaction name="actionAddCategory"/>
    <addaction name="actionAddSubCategory"/>
    <addaction name="actionAddReceipt"/>
    <addaction name="separator"/>
    <addaction name="actionSaveReceipt"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveItem"/>
    <addaction name="actionMoveItem"/>
   </widget>
   <addaction name="menuSubor"/>
   <addaction name="menuRecepty"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPomocn_k"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionViewMode">
   <property name="text">
    <string>Prezera≈• recepty</string>
   </property>
  </action>
  <action name="actionEditMode">
   <property name="text">
    <string>Upravi≈• Recepty</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>O Programe...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Ukonƒçi≈•</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Pomoc</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Vytlaƒçi≈• recept</string>
   </property>
  </action>
  <action name="actionAddCategory">
   <property name="text">
    <string>Prida≈• kateg√≥riu</string>
   </property>
  </action>
  <action name="actionAddReceipt">
   <property name="text">
    <string>Prida≈• recept</string>
   </property>
  </action>
  <action name="actionEditItem">
   <property name="text">
    <string>Uprav polo≈æku</string>
   </property>
  </action>
  <action name="actionSaveReceipt">
   <property name="text">
    <string>Ulo≈æi≈• recept</string>
   </property>
  </action>
  <action name="actionRemoveItem">
   <property name="text">
    <string>Odstr√°ni≈• polo≈æku</string>
   </property>
  </action>
  <action name="actionAddSubCategory">
   <property name="text">
    <string>Prida≈• podkateg√≥riu</string>
   </property>
  </action>
  <action name="actionMoveItem">
   <property name="text">
    <string>Presun√∫≈• polo≈æku</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
